src/new/wallet.js

```diff
@@ -2499,6 +2499,14 @@ class HathorWallet extends EventEmitter {
   async checkPinAndPassword(pin, password) {
     return await this.checkPin(pin) && await this.checkPassword(password);
   }
+
+  /**
+   * Check if the wallet is a hardware wallet.
+   * @returns {Promise<boolean>}
+   */
+  async isHardwareWallet() {
```

Comment:
We need the same method in the wallet service facade and also to update the interface.

Reply:
Done.
But the method in the wallet-service facade will always return `false` since we currently do not support hardware wallets with the wallet-service.